openmdao/core/tests/test_check_derivs.py

```diff
@@ -63,28 +63,63 @@ def compute_partials(self, inputs, partials):
         partials['f_xy', 'x'] = 2.0*x*sc*sc - 6.0*sc + y*sc*sc
         partials['f_xy', 'y'] = 2.0*y*sc*sc + 8.0*sc + x*sc*sc
 
+class MyCompGoodPartials(ExplicitComponent):
+    def setup(self):
+        self.add_input('x1', 3.0)
+        self.add_input('x2', 5.0)
+        self.add_output('y', 5.5)
+        self.declare_partials(of='*', wrt='*')
 
-class TestProblemCheckPartials(unittest.TestCase):
+    def compute(self, inputs, outputs):
+        """ Doesn't do much. """
```

Comment:
I would vote to banish all occurrences of this docstring ;) 
maybe replace it with something like "Calculates y = 3*x1 + 4*x2" in this case...

Reply:
I agree that docstring looks bad. The only issue with your proposed replacement is that it is totally obvious from the code what it does so that doesn't look too great either. 

Could we just use the docstring we use for the compute method from ExplicitComponent? 

"Compute outputs given inputs".

Reply:
I think the docstring being a more readable, non-coded version of the equation is useful.  Just IMO.

Reply:
If the compute docstring says anything, it should say something like "Values for compute are correct" to contrast with the derivatives, which are intentionally declared with incorrect values to trigger the desired response in the checks. We don't care that this component provides calculation for the equation `y = 3*x1 + 4*x2`; it is irrelevant.

Further, if we are going to dig into intra-test object docstrings this much, then we should turn on lint checks for the tests and enforce it all rigidly, and put up a story to clean up the tests.

Reply:
I guess I need Justin or John to give their thoughts on this debate.